recipes/radical.utils/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "radical.utils" %}
+{% set version = "0.47.5" %}
+{% set sha256 = "423c1959c7b387b0a54128a7263d46c6673a247fd4e06de4e728b7c81005d109" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://pypi.io/packages/source/r/{{name}}/{{name}}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+build:
+  number: 0
+  script: python -m pip install .
+  skip: True  # [py<27]
+  skip: True  # [not unix]
+  skip: True  # [not mac]
```

Comment:
This package is mac only?

Reply:
It is unix and mac. So skip applies only once.  Will fix